<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BloodPump</class>
 <widget class="QWidget" name="BloodPump">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border:none;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetContent" native="true">
     <property name="styleSheet">
      <string notr="true">#widgetContent{
	background-image: url(:/public/right/bloodpump_close.png);
}
#bloodPumpPoint{
	background-image:url(:/public/right/bloodlight_open.png);
	background-repeat:no-repeat;
}
#pushButton{
	background-image: url(:/public/right/bloodpump_closeimg.png);
}</string>
     </property>
     <widget class="QWidget" name="bloodPumpPoint" native="true">
      <property name="geometry">
       <rect>
        <x>65</x>
        <y>52</y>
        <width>22</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonShade">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>240</width>
        <height>210</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-top-left-radius: 105px;
border-bottom-left-radius: 105px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="widgetRoViewCon" native="true">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>54</y>
        <width>102</width>
        <height>102</height>
       </rect>
      </property>
     </widget>
     <zorder>widgetRoViewCon</zorder>
     <zorder>bloodPumpPoint</zorder>
     <zorder>pushButtonShade</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
